src/Hl7.Fhir.R5.Core/FhirPath/FhirPathExtensions.cs

```diff
@@ -34,7 +34,7 @@ public static class FhirPathExtensions
             ITypedElement? navResolver(string url)
             {
                 var resource = resolver(url);
-                return resource?.ToTypedElement();
+                return resource?.ToTypedElement(ModelInfo.ModelInspector);
```

Comment:
Since this is code in a satellite, there is now an overload of `ToTypedElement()` *without* the ModelInspector parameter!